ocean_lib/ocean/ocean_assets.py

```diff
@@ -494,7 +494,6 @@ def pay_for_service(
                 amount_base,
                 from_wallet.address,
             )
-            return tx_hash
```

Comment:
This doesn't look like a "test coverage". And I don't see any context / motivation of why to change it.

I recommend putting this in a different PR. (Quickly create a new issue with the motivation, then branch, then put these changes in, merge it to main, then merge from main to this branch.)

Reply:
well, finishOrder is never used and never tested. The change here is just to delay the return until we finish the Order.

Reply:
OK to leave it here. Be sure to mention the "why" in your message in squash merge.

Reply:
please remove `finishOrder`, it is meant to be used by the provider, not the consumer.

Reply:
done!